mslib/msui/_tests/test_mscolab.py

```diff
@@ -357,6 +322,35 @@ def test_delete_project_from_list(self):
         self.window.mscolab.delete_project_from_list(p_id)
         assert self.window.mscolab.active_pid is None
 
+    @mock.patch("PyQt5.QtWidgets.QMessageBox.question", return_value=QtWidgets.QMessageBox.Yes)
+    def test_user_delete(self, mockmessage):
+        pytest.skip("yet to refactor for new UI")
+        self._connect_to_mscolab()
+        self._create_user("something", "something@something.org", "something")
+        self._login("something@something.org", "something")
+        u_id = self.window.user['id']
+        QtTest.QTest.mouseClick(self.window.deleteAccountButton, QtCore.Qt.LeftButton)
+        QtWidgets.QApplication.processEvents()
+        assert len(self.window.listProjects) == 0
+        assert self.window.loggedInWidget.isVisible() is False
+        with self.app.app_context():
+            assert User.query.filter_by(emailid='something').count() == 0
+            assert Permission.query.filter_by(u_id=u_id).count() == 0
+
+    def test_open_help_dialog(self):
+        pytest.skip("yet to refactor for new UI")
```

Comment:
After we merged the gui part, we have to decide if the help can become a tutorial

Reply:
Do you mean "help" could covered by hrithik's project?

Reply:
I think so too.